src/arbiter.rs

```diff
@@ -217,3 +264,52 @@ impl<I: Send, E: Send> Handler<Execute<I, E>> for Arbiter {
         msg.exec()
     }
 }
+
+/// Builder struct for an Arbiter.
+pub struct Builder {
+    /// Name of the Arbiter. Defaults to "actor" if unset.
+    name: Option<String>,
+
+    /// Whether the Arbiter will stop the whole System on uncaught panic. Defaults to false.
+    stop_system_on_panic: bool,
+}
+
+impl Builder {
+    fn new() -> Self {
+        Builder {
+            name: None,
+            stop_system_on_panic: false,
```

Comment:
As I mentioned previously, should we make it true by default?

It is not a problem to change the behavior so suddenly.

Reply:
I am not sure. Let’s wait till next major version

Reply:
Ok, then @Xorlev ignore my comment here 😄